### ISSUE TITLE
Hi, Flacon is now also available for Voidlinux

### ISSUE THREAD

**Issue by MirceaMx** [flacon/flacon#71]
sudo xbps-install flacon - that is how you install it in Voidlinux.
Thank you.

**Comment by SokoloffA** [comment]
Thank you! Is this the official repository? Can I write?

## Void Linux
_Void Linux_ users can install it from the official repositories.
```
    sudo xbps-install flacon
```

**Comment by MirceaMx** [comment]
Yes (it is in the official repository) and yes (you can/should write). Thanks.
